docs/server.rst

```diff
@@ -184,21 +210,38 @@ LDAP authorisation config
 
 ::
 
+ # Limit authentication to only GET and POST requests, auth will not be sent with OPTIONS
  <Location / >
-	AuthType Basic
-	AuthBasicProvider ldap
-	AuthName "LDAP Login For NPG Streaming"
-	AuthLDAPURL "sanger ldap string"
-	Require valid-user
-	AuthLDAPRemoteUserAttribute uid
-	RewriteEngine On
-        RewriteRule .* - [E=PROXY_USER:%{LA-U:REMOTE_USER},NS]
-	RequestHeader set X-Remote-User %{PROXY_USER}e
+   <Limit GET POST>
+     AuthType Basic
+     AuthBasicProvider ldap
+     AuthName "LDAP Login For NPG Streaming"
+     AuthLDAPURL "sanger ldap string"
+     Require valid-user
+     AuthLDAPRemoteUserAttribute uid
+     RewriteEngine On
+     RewriteRule .* - [E=PROXY_USER:%{LA-U:REMOTE_USER},NS]
+     RequestHeader set X-Remote-User %{PROXY_USER}e
+   </Limit>
   </Location>
 
 Reverse proxy configuration
 ---------------------------
 
+If a reverse proxy is set as an entry point for the application, the server will
+need to be aware of the reverse proxy addresses and paths mapped. The list of
```

Comment:
There exists an Apache module that solves this problem for html pages. No such kuck for json content, unless we extend this existing module.